addon/i3dio/ui/object.py

```diff
@@ -872,6 +893,42 @@ def draw(self, context):
         row.prop(obj.i3d_mapping, 'mapping_name')
 
 
+@persistent
+def check_lod_distance(dummy):
+    for obj in bpy.data.objects:
+        if obj.type == 'EMPTY':
+            if hasattr(obj, 'i3d_attributes') and 'lod_distance' in obj['i3d_attributes']:
+                current_lod = obj['i3d_attributes']['lod_distance']
+                # Skip if already converted
+                if not isinstance(current_lod, str):
+                    continue

```

Comment:
If the property is renamed to `lod_distances` instead, then this check can go away as there will be no clash anymore